http/server.go

```diff
@@ -54,14 +54,15 @@ func (s *Server) Run(done <-chan struct{}) error {
 		}
 	}()
 
-	fmt.Printf("Server started\n")
+	// TODO replace with logging
+	// fmt.Printf("Server started\n")
 
 	select {
 	case err := <-errors:
 		return err
 	case <-done:
-		fmt.Printf("Server stopped\n")
-
+		// TODO replace with logging
```

Comment:
will this be done in this PR? If not, we can  keep the fmt logs for now if they are useful (otherwise I agree let's remove/comment them out)

Reply:
Replaced this with logging.